crypto/src/conversation.rs

```diff
@@ -68,20 +72,25 @@ pub struct MlsConversation {
     configuration: MlsConversationConfiguration,
 }
 
+/// It is a wrapper for the welcome and the message returned when creating a group/conversation
```

Comment:
```suggestion
/// Returned when initializing a conversation. Different from conversation created from a [`Welcome`] message or an external commit.
```
Also document members